packages/office-ui-fabric-react/src/components/Calendar/CalendarDay.tsx

```diff
@@ -315,7 +316,6 @@ export class CalendarDay extends BaseComponent<ICalendarDayProps, ICalendarDaySt
                           })}
                           role={'button'}
                           onKeyDown={this._onDayKeyDown(day.originalDate, weekIndex, dayIndex)}
-                          onClick={day.isInBounds ? day.onSelected : undefined}
```

Comment:
Don't know for sure but it seems like a button without an onclick might cause problems for keyboarding. Could you put the click handler on both the button and the td (possibly with a wrapper function to ensure propagation is stopped)?

Reply:
I tested with that, and because the onClick doesn't catch the Event and .stopPropagation(), that actually results in the event firing twice.

Because we don't stop propagation, when you fire using either Enter or by using Space on the keyboard, that event bubbles up from the <button> up to the cell, so it still gets invoked. I agree it looks wonky in the code though.

Reply:
Testing right now with the wrapper function, should work fine.

Reply:
seems to work, updated